lib/code_corps/github/events/installation.ex

```diff
@@ -13,5 +24,89 @@ defmodule CodeCorps.GitHub.Events.Installation do
   - do the work
   - marked the passed in event as "processed" or "errored"
   """
-  def handle(%GithubEvent{}, %{}), do: :not_fully_implemented
+  @spec handle(GithubEvent.t, map) :: {:ok, GithubEvent.t}
+  def handle(%GithubEvent{action: "created"} = event, payload) do
+    event
+    |> start_event_processing()
+    |> do_handle(payload)
+    |> stop_event_processing(event)
```

Comment:
The "processing" state seems redundant here. Sure, the processing action is slightly longer lasting that most things we do, but this is all called in succession, as part of one large async process, not as multiple async processes. It basically just adds an extra db write.

We never really call the handler for an existing event so it's not like we can match against `state: "processing"` her and abort if that is  the case.

Right now, our flow is always  "create event as unprocessed -> mark as processing -> handle payload -> mark as processed". Everything except the first step is done as a single async block, so it's not really doing us any good as it is right now. Feels like we're implementing for some future thing that might happen, may even likely happen, but we did a similar thing with stripe and it did not happen yet in that case.

With stripe, there's at least the added argument of having unique event ids. With github, all we have are delivery ids, which are different with each webhook request.